rock_paper_scissors.py

```diff
@@ -150,8 +150,8 @@ def main():
     game.rounds = int(input('How many odd rounds would you like to play? '))
 
     while not game.is_complete:
-        user.choice = input('Choose rock, paper, or scissors: ')
-        computer.choice = random.choice(VALID_CHOICES)
+        user.choice = input('Choose from the following: {}: '.format(', '.join(game.valid_choices)))
```

Comment:
I prefer #1, it's easier to read :)

@TylerHendrickson I was unsure how to include an 'or' for the second to last string without creating another function. Currently it prints: `Choose from the following: rock, scissors, paper:` Is this another issue we can add?